django/core/management/__init__.py

```diff
@@ -118,17 +119,29 @@ def call_command(command_name, *args, **options):
     }
     arg_options = {opt_mapping.get(key, key): value for key, value in options.items()}
     parse_args = [str(a) for a in args]
+
+    # Return parser actions and sub parser choices actions.
+    def get_actions(parser):
+        actions = []
+        for opt in parser._actions:
+            actions.append(opt)
```

Comment:
Is this required? I think we should collect parsers and choices from sub-parsers, e.g.
```python
if isinstance(opt, _SubParsersAction):
    for sub_opt in opt.choices.values():
        actions += get_actions(sub_opt)
else:
    actions.append(opt)
```
After that we can remove `if opt.option_strings` from `parse_args += [...`

Reply:
Yes, I think it is required, we should collect action of sub-parsers choices sub-parsers (nested sub-parsers). I added a test for it. `subparser2`.

Reply:
If we skip sub-parsers as actions then all tests still pass and `if opt.option_strings` condition is not required anymore, e.g.
```python
# Parser actions and actions from sub-parser choices.
def get_actions(parser):
    for opt in parser._actions:
        if isinstance(opt, _SubParsersAction):
            for sub_opt in opt.choices.values():
                yield from get_actions(sub_opt)
        else:
            yield opt

parser_actions = list(get_actions(parser))
```
That's because `dest` for sub-parsers is not a valid option and don't need to be in `dest_parameters`.

Reply:
Thanks for the suggestion. I changed the patch by your changes.